src/NHibernate/Driver/SqlClientDriver.cs

```diff
@@ -283,6 +287,18 @@ protected static bool IsBlob(DbParameter dbParam, SqlType sqlType)
 			return (sqlType is BinaryBlobSqlType) || ((DbType.Binary == dbParam.DbType) && sqlType.LengthDefined && (sqlType.Length > MsSql2000Dialect.MaxSizeForLengthLimitedBinary));
 		}
 
+		/// <summary>
+		/// Interprets if a parameter is a character (for the purposes of setting its default size)
+		/// </summary>
+		/// <param name="dbParam">The parameter</param>
+		/// <param name="sqlType">The <see cref="SqlType" /> of the parameter</param>
+		/// <returns>True, if the parameter should be interpreted as a character, otherwise False</returns>
+		protected static bool IsChar(DbParameter dbParam, SqlType sqlType)
+		{
+			return (DbType.StringFixedLength == dbParam.DbType || DbType.StringFixedLength == dbParam.DbType) &&
```

Comment:
I guess one of these should be `Ansi` prefixed, shouldn't it?

Reply:
Yes, corrected.